package.json

```diff
@@ -23,6 +23,7 @@
     "eslint": "^4.10.0",
     "eslint-config-prettier": "^2.7.0",
     "eslint-plugin-prettier": "^2.3.1",
+    "eslint-plugin-self": "^1.0.1",
```

Comment:
should update the lockfile?

Reply:
Yes! I was using `npm` so didn't see that.